src/webui/webapplication.cpp

```diff
@@ -372,9 +383,11 @@ void WebApplication::doProcessRequest()
             }
             break;
         case QMetaType::QString:
-        default:
             print(result.data.toString(), Http::CONTENT_TYPE_TXT);
             break;
+        default:
+            status(204);
```

Comment:
It seems that this change is not required by this PR. I would prefer to see it provided as a separate PR, because it affects the behavior of existing endpoints, which will require appropriate care (e.g. changes in documentation).

Reply:
I will move this to a separate PR.

Reply:
#21349